app/models/preorder_information.rb

```diff
@@ -109,7 +109,14 @@ def will_need_chromebooks?
     will_need_chromebooks == 'yes'
   end
 
+  def will_not_need_chromebooks?
+    will_need_chromebooks == 'no'
+  end
+
   def chromebook_information_complete?
+    # do not want to hold them up being able to order
+    return true if school&.la_funded_place_establishment_type?
```

Comment:
```suggestion
    return true if school.la_funded_place_establishment_type?
```

We can safely assume that there will be a school on this model.

Reply:
There was a whole bunch of model specs that failed because they weren't properly set up with a school. Perhaps I shouldn't be lazy and fix all the specs!

Reply:
On further investigation I've left it in. The `chromebook_information_complete?` method gets called from `infer_status` which itself gets called via callbacks during the creation of objects in specs. Removing the `&` causes 400+ specs to fail.